### PR TITLE
Feature/map action fix

### PR DESCRIPTION
**Pull request by jfrankl**
This styles the events and new reservation map pages. The action bar will appear fixed on the bottom of the page on mobile and in the sidebar on desktop. Also, for the event page, I needed to add some javascript to put a class on a container element based on selected tabs. There might be a better way to achieve that goal.

### CODE REVIEW

**Comment by jfrankl** [comment]
image

image

image

image

image

**Comment by maurizi** [comment]
I would squash these two commits together.

I think the JS could use some work, but one of the developers can fix it up at a later time (can you put a `TODO` comment above it?).

**Comment by maurizi** [comment]
Also, you should make a note in the commit message that this fixes #367

File: src/nyc_trees/sass/partials/_map.scss [hunk 1]
```diff
@@ -17,6 +47,16 @@
   height: 400px;
 }
 
+.map-event {
+  // margin-right: -$grid-gutter-width/2;
+  // margin-left: -$grid-gutter-width/2;
```
Comment (maurizi) [inline src/nyc_trees/sass/partials/_map.scss]: You should remove this commented out CSS

**Comment by jfrankl** [comment]
Thanks for the feedback Mike. This should be good to go.

**Comment by maurizi** [comment]
I opened #412 to track the issue we discussed on the event list page.

**Pull request merged**